packages/ui/src/components/shared/settings/VerifyingPassword.tsx

```diff
@@ -39,7 +47,7 @@ const VerifyingPassword: FC<Props> = () => {
 
   return (
     <>
-      <DialogContentText className='mt-4 mb-4'>{t<string>('Enter your wallet password to continue')}</DialogContentText>
+      <DialogContentText className='mt-4 mb-2'>{t<string>('Enter your wallet password to continue')}</DialogContentText>
       <form onSubmit={doVerify} noValidate>
         <TextField
           type='password'
```

Comment:
Let's change the label `'Your wallet password'` to `Wallet password` since we're using `Wallet password` in a lot of places. Introducing a new label (`Your wallet password`) with the same meaning sometimes makes it harder for us to choose which one to use.